tests/acceptance/common.py

```diff
@@ -228,8 +228,8 @@ def determine_active_passive_part(bitbake_variables):
     elif mount_output.find(b) >= 0:
         return (b, a)
     else:
-        raise Exception("Could not determine active partition. Mount output: %s"
-                        % mount_output)
+        raise Exception("Could not determine active partition. Mount output:\n {}" \
+                        "\nwas looking for {}".format(mount_output, (a, b)))
```

Comment:
IIRC `.format()` is supposed to be a preferred way of string formatting. There are some nice tricks you can do with format that you cannot do with `%`.